dev/bin/interaction/environment-example

```diff
@@ -2,8 +2,14 @@
 # It can be used to configure required variables for your local environment
 # without the need of exporting them every time, like offline tokens etc.
 # If you want to use it, make a copy of this named "environment" in this same folder.
+# The dev and stable environment use sso.redhat.com. So if you want to target such environments, set TARGET_ENVIRONMENT=remote and the
+# variable OPENSHIFT_OFFLINE_TOKEN accordingly (retrieve the token from https://console.redhat.com/openshift/token).
+# If you target your local environment, then replace the variables KEYCLOAK_USERNAME and KEYCLOAK_PASSWORD.
```

Comment:
Is it possible to configure also the local instance to use an offline token? I would like to keep the configuration aligned

Reply:
the developer has to edit this file in any case. If the developer has to extract the offline token from the local keycloak instance he has to run an additional command with the username and the password, while in this script he can directly use the username and the password. 
It's a dev script at the end, but if you want we can do it

Reply:
My main concern is, if I have understood correctly, that based on `TARGET_ENVIRONMENT` value, the scripts are going to use `OPENSHIFT_OFFLINE_TOKEN` or `KEYCLOAK_USERNAME`/`KEYCLOAK_PASSWORD`.
In general if we change the configuration to use the token also locally I think we should be able to get rid of all of this and keep two variables only: `SSO_REDHAT_URL` and `OPENSHIFT_OFFLINE_TOKEN` (no need to specify the target env nor keycloak variables).
Same for the if/else in `get-token.sh`

I don't know how hard is to do this different configuration so I'm fine to proceed with this PR and do a followup ticket for this cleanup (if we agree to do it).